docs/userguide/networking/default_network/build-bridges.md

```diff
@@ -0,0 +1,73 @@
+<!--[metadata]>
++++
+title = "Build your own bridge"
+description = "Learn how to build your own bridge interface"
+keywords = ["docker, bridge, docker0, network"]
+[menu.main]
+parent = "smn_networking_def"
++++
+<![end-metadata]-->
+
+# Build your own bridge
+
+The information in this section explains building your own bridge to replace within the Docker default bridge. This is a `bridge` network named
+`bridge` created automatically when you install Docker.  
+
+**Note**: The [Docker networks feature](../dockernetworks.md) allows you to create user-defined networks in addition to the default bridge network.
+
+
+You can set up your own bridge before starting Docker and use `-b BRIDGE` or `--bridge=BRIDGE` to tell Docker to use your bridge instead.  If you already have Docker up and running with its default `docker0` still configured, you will probably want to begin by stopping the service and removing the interface:
```

Comment:
Here again. Except for the framing, I haven't touched the interior content of the original piece. I'll do the wraps but I have no scope to restructure the material. I know it is bad; please feel free to file an issue.